package.json

```diff
@@ -12,13 +15,42 @@
     "url": "https://github.com/Krombik/html2react/issues"
   },
   "homepage": "https://github.com/Krombik/html2react#readme",
-  "type": "module",
-  "source": "src/index.ts",
   "engines": {
     "node": ">=10"
   },
+  "type": "module",
+  "source": "src/index.ts",
+  "main": "./build/index.cjs",
+  "module": "./build/index.js",
+  "types": "./build/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./build/index.js",
+      "default": "./build/index.cjs",
+      "types": "./build/index.d.ts"
+    },
+    "./HTML2React": {
```

Comment:
If you want to support patterns like this:

```ts
import HTML2React from 'react-html-string-parser/HTML2React'
```

You can use `exports` mapping to provide virtual routes to deep built files in a more ergonomic way.